source/_integrations/icloud.markdown

```diff
@@ -84,11 +99,32 @@ If 2 Step Authentication is enabled for your iCloud account. The integration wil
 
 Go into your Home Assistant configuration `.storage` folder and delete the "icloud" folder, then retry.
 
+## Platforms
+
+### Device Tracker
+
+The iCloud integration will track available devices on your iCloud account.
+
+### Sensor
+
+The iCloud integration will add a battery sensor for each iCloud devices available on your iCloud account.
+
 ## Services
 
 4 services are available:
 
-- **update**: This service can be used to ask an update of a certain iDevice or all devices linked to an iCloud account. Request will result in new Home Assistant [state_changed](/docs/configuration/events/#event-state_changed) event describing current iPhone location. Can be used in automations when manual location update is needed, e.g., to check if anyone is home when door's been opened.
-- **play_sound**: This service will play the Lost iPhone sound on your iDevice. It will still ring if you are on "Mute" or "Do not disturb" mode.
-- **display_message**: This service will display a message on your iDevice. It can also ring your device.
-- **lost_device**: This service will put your iDevice on "lost" mode (compatible devices only). You have to provide a phone number with a suffixed [country code](https://en.wikipedia.org/wiki/List_of_country_calling_codes) and a message.
+### Service `icloud.update`
+
+This service can be used to ask an update of a certain iDevice or all devices linked to an iCloud account. Request will result in new Home Assistant [state_changed](/docs/configuration/events/#event-state_changed) event describing current iPhone location. Can be used in automations when manual location update is needed, e.g., to check if anyone is home when door's been opened.
```

Comment:
```suggestion
This service can be used to ask an update of a certain iDevice or all devices linked to an iCloud account. Request will result in new Home Assistant [state_changed](/docs/configuration/events/#event-state_changed) event describing current iPhone location. It can be used in automations when a manual location update is needed, e.g., to check if anyone is home when a door been opened.
```